app/views/audits/allocations/_toolbar.html.erb

```diff
@@ -1,31 +1,15 @@
 <div class="allocation-menu">
   <div class="row">
-    <div class="col-sm-7">
-      <div class="checkbox">
-        <label>
-          <%= check_box_tag :select_all, nil, false %> Select all
-        </label>
-      </div>
-      <% if select_all_pages > 1 %>
-        <div class="checkbox if-js-hide" id="select_all_pages_container">
-          <label>
-            <%= check_box_tag :select_all_pages, nil, false %>
-            Select <%= number_with_delimiter(select_all_count) %> items
-            on all pages
-          </label>
-        </div>
-      <% end %>
-    </div>
-    <div class="col-sm-5">
-      <div class="form-group form-inline pull-right">
-        <%= label_tag :allocate_to, "Assign to" %>
-        <%= select_tag "allocate_to",
-                       allocation_options_for_select(params[:allocate_to]),
-                       class: "form-control",
-                       id: "allocate_to" %>
-        <%= filter_to_hidden_fields(:allocate_to) %>
-        <%= submit_tag "Go", class: "btn btn-default" %>
-      </div>
+    <div class="form-group form-inline pull-left">
+      <%= label_tag :allocate_to, "Assign", class: "allocate" %>
+      <%= text_field_tag :batch_size, "", class: "form-control" %>
```

Comment:
This needs a label to be accessible.

Reply:
I have created a [Trello][1] card to address this issue. I think it is better to split the card in the same way we have done for the multiple organisation selection.

[1]: https://trello.com/c/NVzzXbUL/529-make-batch-selection-accessible

Reply:
We know this is fundamentally not accessible, with the multiple organisation selection we made our best efforts to make it accessible and wrote a card to look into it further.

If we want to get this merged we should think about a simpler solution, with a conventional form structure and then write a card to explore a more sophisticated approach that remains accessible.